doc/installing.md

```diff
@@ -34,7 +34,7 @@ In order to successfully build and install hipSYCL, the following major requirem
 * *For the CPU backend*: Any C++ compiler with **OpenMP** support should do.
 * python 3 (for the `syclcc` and `syclcc-clang` compiler wrappers)
 * `cmake`
-* hipSYCL also depends on HIP, but already comes bundled with a version of the HIP headers for the CUDA backend. The ROCm backend will use the HIP installation that is installed as part of ROCm.
+* HIP (the ROCm backend will use the HIP installation that is installed as part of ROCm).
```

Comment:
This should probably be moved to *For the ROCm backend* anyway.. Recent hipSYCL only needs HIP when targeting AMD.

Reply:
Could be just removed then, as HIP is part of ROCm, so only ROCm is required?

Reply:
Hm.. It's a component of ROCm. I'm not sure we can assume every ROCm installation is a *full* installation. There are some packages in some distributions that only install ROCm's OpenCL for example. So we could list as requirement something like *ROCm, in particular HIP*